### DISCUSSION TITLE
How to disable snapshots - Proxmox Updater

### DISCUSSION THREAD

**Discussion by JDC8077** [BassT23/Proxmox#111] [Q&A] [answered]
I have my config file setup to disable backups, which is done seperately as weekly job. Each time the "update" is ran, it trys to/runs a snapshot prior to installing updates. Is there a way to disable this, similar to backups when updating? Thanks

**Comment by BassT23** [comment, marked as answer]
If you want, you can tryout the latest beta:

`update beta -up`

Please be aware; because of name and directory changing, you need an reboot of the node, after update.
For now there are a bug in the latest version.
[look here](https://github.com/BassT23/Proxmox/issues/110)
Hopefully I can fix it as soon as possible
